SQL/0000-00-00-schema.sql

```diff
@@ -206,6 +206,7 @@ CREATE TABLE `users` (
   `State` varchar(255) default NULL,
   `Zip_code` varchar(255) default NULL,
   `Country` varchar(255) default NULL,
+  `Phone` varchar(15) default NULL,
```

Comment:
Is this line supposed to be changed? It looks like the result of an errant rebase.